src/main/java/org/lightcouch/View.java

```diff
@@ -438,14 +440,20 @@ public View key(Object... key) {
      *                 keys.
      */
     public View startKey(Object... startKey) {
-        this.startKey = getKeyAsJson(startKey);
-        uriBuilder.query("startkey", this.startKey);
+        this.startKey = getKeyAsJsonElement(startKey);
+        uriBuilder.query("startkey", this.startKey, true);
+        return this;
+    }
+
+    public View startKeyDocId(String startKeyDocId, boolean replace) {
```

Comment:
For the query URIs built by the `View` class I can see no cases where appending rather than replacing a parameter would be appropriate so we should always just use `true` to replace in this class.